crates/red_knot_python_semantic/src/semantic_index/definition.rs

```diff
@@ -153,6 +160,17 @@ pub(crate) struct ComprehensionDefinitionNodeRef<'a> {
     pub(crate) first: bool,
 }
 
+#[derive(Copy, Clone, Debug)]
+pub(crate) struct MatchPatternDefinitionNodeRef<'a> {
+    /// The outermost pattern node in which the identifier being defined occurs.
+    pub(crate) pattern: &'a ast::Pattern,
```

Comment:
Maybe `pattern_root` might be a better name for this field? (And same for the equivalent field on `MatchPatternDefinitionKind`?)

Reply:
It is a pattern, so I think `root_pattern` would be better than `pattern_root` if we did change the name. But tbh I think `pattern` is also fine; typically we'd call everything else a "sub-pattern" -- the top-level pattern is "the pattern."